src/tests/src/solver/simulation/remix-storage/test-hydro-remix.cpp

```diff
@@ -14,110 +15,139 @@
 
 using namespace Antares::Solver::Simulation;
 
+struct HydroStorageData
+{
+    HydroStorageData(size_t size)
+    {
+        hydroGen.assign(size, 0.);
+        levels.assign(size, 0.);
+        inflows.assign(size, 0.);
+    }
```

Comment:
Probably doesn't matter, but prefer member initialization
```suggestion
    HydroStorageData(size_t size) :
    hydroGen(size, 0.),
    levels(size, 0.),
    inflows(size, 0.)
    {
    }
```
this way you call this constructor `std::vector<T>::vector(std::size_t, const T&)` directly instead of `std::vector<T>::vector()` then `assign`.